app/models/review.rb

```diff
@@ -92,8 +92,8 @@ def assigner_is_a_grant_editor
     errors.add(:assigner, :may_not_add_review) unless Pundit.policy(assigner, grant).grant_editor_access?
   end
 
-  def reviewer_is_not_applicant
-    errors.add(:reviewer, :may_not_review_own_submission) if reviewer == submission.applicant
+  def reviewer_is_not_submitter
+    errors.add(:reviewer, :may_not_review_own_submission) if reviewer == submission.submitter
```

Comment:
see previous comment re applicant/submitter. i think the rules here need to be well thought out given the examples we've encountered here so far.